docs/api.rst

```diff
@@ -506,7 +506,31 @@ Converters
 Deprecated APIs
 ---------------
 
+.. _version-info:
+
+To help you writing backward compatible code that doesn't throw warnings on modern releases, the ``attr`` module has an ``__version_info__`` attribute as of version 19.2.0.
```

Comment:
"To help you write backward compatible code" I think sounds better.

Reply:
Ugh you're right. Sometimes I wonder what I was thinking.